src/file-events/cpp/win_fsnotifier.cpp

```diff
@@ -197,7 +207,7 @@ void Server::handleEvent(JNIEnv* env, const u16string& path, FILE_NOTIFY_INFORMA
         }
     }
 
-    log_fine(env, "Change detected: 0x%x '%s'", info->Action, utf16ToUtf8String(changedPath).c_str());
+    // log_fine(env, "Change detected: 0x%x '%s'", info->Action, utf16ToUtf8String(changedPath).c_str());
```

Comment:
Is this commented out accidentally?

Reply:
No, but it shouldn't be merged. I'll add a TODO.

Reply:
OK. Let's handle it when we tackle better logging.